cookiecutter-xblock/{{cookiecutter.repo_name}}/manage.py

```diff
@@ -13,7 +13,7 @@
 if __name__ == "__main__":
     os.environ.setdefault(
         "DJANGO_SETTINGS_MODULE",
-        "{{cookiecutter.package_name}}.locale.settings"
+        "{{cookiecutter.package_name}}.conf.locale.settings"
```

Comment:
@shadinaif Is this mandatory? Usually the `locale` dir isn't expected to have any Python code.

My expectation is to find the setting in `{{cookiecutter.package_name}}.dev_settings` or `{{cookiecutter.package_name}}.locale_settings` if you must.

I understand that this pattern is already set, but it appears to be incorrect. Not a blocker though.

Reply:
@shadinaif please take a look at this note.

Reply:
I used `.dev_settings` not `{{cookiecutter.package_name}}.dev_settings`, to place the file next to `manage.py` (which uses the settings file)